ichnaea/log.py

```diff
@@ -72,12 +87,39 @@ def configure_logging():
                 "handlers": handlers,
                 "level": logging_level,
             },
+            # https://stripe.com/blog/canonical-log-lines
+            "canonical-log-line": {
```

Comment:
I like using the Stripe blog post as a "why" document for this effort, and the search results for `canonical-log-line` are useful. Still, I'm not sure that I'm justified to add a new logger, and I'll go back to using `ichnaea` if desired.